libbpf-tools/xfsslower.h

```diff
@@ -11,14 +11,14 @@
 #define TRACE_FSYNC  'F'
 
 struct event {
-	char file[DNAME_INLINE_LEN];
-	char task[TASK_COMM_LEN];
 	__u64 delta_us;
 	__u64 end_ns;
 	__s64 offset;
 	ssize_t size;
 	pid_t tgid;
 	char type;
+	char file[DNAME_INLINE_LEN];
+	char task[TASK_COMM_LEN];
```

Comment:
this actually might be harmful, as it puts file and task at non-aligned offset. Given both file (32 bytes) and task (16 bytes) have 8-aligned sizes, it's ok to keep them at the bottom. Please revert this part. It's similar for other places with multiple-of-8-sized arrays, they are not going to harm alignment.

Reply:
Thanks @anakryiko , I've revert them back to ensure that all fields of the structure at 8-aligned offset if they have 8-aligned sizes.